### PR TITLE
Lev 296

### PR DESCRIPTION
**Pull request by aarongill1**

### CODE REVIEW

File: controllers/OrganisationsController.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+const BaseController = require('./BaseController');
+const { orgLookup } = require('../services/OrganisationSearchService');
+
+class OrganisationsController extends BaseController {
```
Comment (ashley-bickerstaff-homeoffice) [inline controllers/OrganisationsController.js:4]: Could we structure these into a subfolder for "UserManagement"? Just so it's a bit more obvious that this is not really related to the primary function of LEV

Comment (ashley-bickerstaff-homeoffice) [inline controllers/OrganisationsController.js:4]: Looking at the other controllers, it looks like it's a one-to-one relationship with routes. If we need to have another controller for when we come to list all organisations (the LEV admin view), I think this needs to be the singular form "organisation".

File: locales/en/default.json [hunk 2]
```diff
@@ -159,6 +159,13 @@
         "pageTitle": "User Activity Report results"
       }
     },
+    "user-management": {
+      "serviceName": "Administration",
+      "organisationsPage": {
```
Comment (ashley-bickerstaff-homeoffice) [inline locales/en/default.json:164]: Similar to one of my previous comments, I think this needs to be singular, rather than pluralised at this point:

"organisationPage"
"Manage organisation"
"Organisation"

File: locales/en/default.json [hunk 3]
```diff
@@ -159,6 +159,13 @@
...
+        "title": "Manage organisations",
```
Comment (ashley-bickerstaff-homeoffice) [inline locales/en/default.json:165]: Is it possible to use placeholder variables here so this could be "Manage organisation: ACME inc"?

File: models/OrganisationRestApiModel.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const HmpoModel = require('hmpo-model');
+const { iamApi } = require('../config');
+const baseUrl = `${iamApi.protocol}://${iamApi.host}:${iamApi.port}`;
```
Comment (ashley-bickerstaff-homeoffice) [inline models/OrganisationRestApiModel.js:5]: Is there a way we can abstract this somewhere? I imagine we'll need this in quite a few places

File: models/OrganisationRestApiModel.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+class OrganisationRestApiModel extends HmpoModel {
+
+  requestConfig(config, args) {
+    const retConfig = Object.assign({}, config);
```
Comment (ashley-bickerstaff-homeoffice) [inline models/OrganisationRestApiModel.js:10]: Should this be "reqConfig", rather than ret?

File: routes/organisations/index.js [hunk 6]
```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+const wizard = require('hmpo-form-wizard');
+const steps = require('./steps');
+
+module.exports = wizard(steps, null, {
```
Comment (ashley-bickerstaff-homeoffice) [inline routes/organisations/index.js:6]: Just another note on pluralisation here so it doesn't get missed if we change it

**Review by ashley-bickerstaff-homeoffice — CHANGES_REQUESTED** [review]

File: controllers/OrganisationsController.js (hunk 1 above)
Comment (aarongill1) [reply controllers/OrganisationsController.js:4]: UserManagement/ directory added in Controllers/ and Services/ and move relevant files.

Comment (aarongill1) [reply controllers/OrganisationsController.js:4]: Done

File: locales/en/default.json (hunk 2 above)
Comment (aarongill1) [reply locales/en/default.json:164]: Done

File: models/OrganisationRestApiModel.js (hunk 5 above)
Comment (aarongill1) [reply models/OrganisationRestApiModel.js:10]: Can potentially be changed but the function this is overriding uses retConfig in the framework:

https://github.com/HMPO/hmpo-model/blob/master/lib/remote-model.js#L70

So I haven't changed but can do if required

File: routes/organisations/index.js (hunk 6 above)
Comment (aarongill1) [reply routes/organisations/index.js:6]: Done

File: views/pages/organisations/index.html [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+{% set govukServiceUrl = "/organisations" %}
+
+{% block mainContent %}
+    {{ super() }}
+
+{% if orgInfo.name == undefined %}
+<h2>Organisation does not exist</h2>
+{% endif %}
+
+<h2>{{ orgInfo.name }}</h2>
```
[the remark this answers is not in the document]
Comment (aarongill1) [reply views/pages/organisations/index.html:16]: Yeah just doesn't  appear - it's more placeholder content for now anyhow. 

Probably worth noting that the iam-api returns a 400 when the org doesn't exist and we intercept it as an error in the ui...if we wanted do, we could and assuming we could render an error page instead of conditional text if required...

File: locales/en/default.json (hunk 3 above)
Comment (aarongill1) [reply locales/en/default.json:165]: Done one better - variable comes from orgInfo retrieved, so is dynamic.

File: models/OrganisationRestApiModel.js (hunk 4 above)
Comment (aarongill1) [reply models/OrganisationRestApiModel.js]: Done

File: models/OrganisationRestApiModel.js (hunk 5 above)
Comment (ashley-bickerstaff-homeoffice) [reply models/OrganisationRestApiModel.js:9]: Nah. From what you've said, it sounds like this if the convention and that trumps a minor rename IMO

**Review by ashley-bickerstaff-homeoffice — APPROVED** [review]
